src/app/global/user_profile.nim

```diff
@@ -15,8 +15,7 @@ QtObject:
     preferredName: string
     thumbnailImage: string
     largeImage: string
-    userStatus: bool
-    #currentUserStatus: int
+    currentUserStatus: int
```

Comment:
I used int here because it is exposed to Qml. I was not sure if enum will work.

Reply:
It's exposed through a property getter, so it could be of any type.

```diff
   proc getCurrentUserStatus*(self: UserProfile): bool {.slot.} =
-    self.currentUserStatus
+    self.currentUserStatus.int
```